packages/lesswrong/components/vulcan-forms/Form.tsx

```diff
@@ -1085,6 +1085,7 @@ class Form<T extends DbObject> extends Component<any,any> {
     revertLabel: this.props.revertLabel,
     cancelCallback: this.props.cancelCallback,
     revertCallback: this.props.revertCallback,
+    submitForm: this.submitForm,
```

Comment:
Do we still need this change?

Reply:
It's not strictly necessary for this PR, but I do think this was a fairly obvious thing to have passed in.

Reply:
It's not technically needed for this PR, but I do think it's pretty silly for the submitFormContext to not actually provide the submitForm and would prefer to leave it in.